### DISCUSSION TITLE
Code changes overwritten by Swell app

### DISCUSSION THREAD

**Discussion by mewa** [swellstores/community#34] [Help] [answered]
Hi all,

I bumped into this behaviour on several occasions now and I'd love to find out what's the recommended behaviour for getting around this.

I can't seem to figure out how to make persistent changes to the app's code (in my case, the origin theme, storefront hosted on Swell). Some of them seem to go through without any issues (e.g. files under `pages/`).

The other (e.g. `static/` and `store/`)  are often overwritten after pushing them to the git remote (a follow-up commit showing up as "merge" of the upstream branch). I say often, because I have 2 shops and the same patch got overwritten in one of them, but not the other one.

Does anyone have a workaround for it?

Thanks!

-----

EDIT:
For reference, here's what it looks like when fetching changes from git:
```
$ git fetch
From swell.store:wasabi-hq-eu
 + c38c038...b69a535 production -> origin/production  (forced update)
 ```
 commit `c38c03815a62b00d9de44750b575f7d53ef5f773` is 8 commits ahead and has been pushed before.

**Comment by rodrigoddalmeida** [comment]
Hi, @mewa 

Have you tried changing `version` in `package.json`? That'll signal the build process to do a full build. Here's additional documentation on it: https://developers.swell.is/storefronts/update-swell-themes-with-git#triggering-a-new-build

**Reply by mewa** [reply]
Thanks for your response @rodrigoddalmeida. I tried your suggestion and while it does trigger a full build it doesn't help with the changes being overridden.

There seems to be another git upstream, which gets out of sync and forcefully merges the changes. Moreover, my change of the package version also got reverted once I  published an update (to the website's footer menu) from the store's dashboard.

**Reply by logeshswell** [reply, marked as answer]
@mewa There was a configuration found missing in your store.
I just added them and updated the code changes with the latest origin theme. After this update, I didn't notice any issues with overwriting the existing code changes and hope it is working fine as expected now.
Let me know in this thread if you notice any issues later.
